packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -560,6 +541,13 @@ abstract class PageTransitionsBuilder {
   /// const constructors so that they can be used in const expressions.
   const PageTransitionsBuilder();
 
+  /// Provideds a secondary transition to the previous route.
+  ///
+  /// {@macro flutter.widgets.delegatedTransition}
+  ///
+  /// If left as null, the previous page will not animate.
```

Comment:
Is this true? won't they be using their own transition?

Reply:
You're right, this is incorrect.